app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Api::V1::Admin::UsersController < Api::V1::BaseController
+  before_action :authenticate_user!
+  before_action :authorize_user
+  before_action :load_user, except: [:index, :roles, :statuses]
+  after_action :verify_authorized
+
+  def index
+    @users = User.all
+    respond_with @users
+  end
+
+  def roles
```

Comment:
I've added a comment to the previous PR. I didn't comment such things before, but I think it is time now to learn to build more REST applications. Try to find some articles on this topic.